source/includes/administration/_environments.md

```diff
@@ -215,34 +227,14 @@ curl -X POST "https://cloudmc_endpoint/v2/environments" \
 }
 ```
 
-Create a new environment in a specific service and organization. You will need the `Environments create` permission to execute this operation.
-
-Required | &nbsp;
--------- | -----------
-`name`<br/>*string* | The name of the new environment. Should be unique in the environment and only contain lower case characters, numbers, dashes and underscores.
-`serviceConnection`<br/>*[ServiceConnection](#administration-service-connections)* | The service connection that the environment should be created in<br/>*required*: `id`
-
-Optional | &nbsp;
--------- | -----------
-`description`<br/>*string* | The description of the new environment.
-`organization`<br/>*[Organization](#administration-organizations)* | The organization that the environment should be created in. *Defaults to your organization*<br/>*required*: `id`
-`membership`<br/>*string* | Type of membership of the environment. ALL_ORG_USERS will add every user in the organization to this environment with the default role. MANY_USERS will allow you to  choose the users you want in the environment and assigned them specific roles. *Defaults to MANY_USERS*
-`roles`<br/>*Array[[Role](#administration-roles)]* | The roles of the environment and the users assigned to them. Also, defines the default role of the environment.<br/>*required*: `name`, `users.id`<br/>*optional*: `isDefault`
-
-Response | &nbsp;
----------- | -----------
-`taskId`<br/>*UUID* | The id of the task
-`taskStatus`<br/>*string* | The status of the task
-`data`<br/>*[Environment](#administration-environments)* | The information about the created environment
-
-The responses' `data` field contains the updated [environment](#administration-environments).
-
 
 <!-------------------- UPDATE ENVIRONMENT -------------------->
 
 
 ### Update environment
 
+Environments are updated asynchronously on the underlying service. When updating an environment any underlying actions performed by the plugin is done asynchronously to the update of the model. The state of the environment will remain the PROVISIONED state. The task returned in the response can be used to track the progress of the asynchronous update. 
```

Comment:
`will remain the PROVISIONED state` --> will remain **in** the PROVISIONED state